draft-ietf-gnap-core-protocol.md

```diff
@@ -2834,17 +2897,16 @@ https://server.example.com/tx
 ~~~
 
 The party then hashes this string with the appropriate algorithm
-based on the "hash_method" parameter under the "finish" key. The resulting
+based on the "hash_method" parameter under the "finish" key of the [interaction finish request](#request-interact-finish). The resulting
 byte array from the hash function is then encoded using URL-Safe Base64
 with no padding {{!RFC4648}}. The resulting string is the hash value.
 
-The "hash_method" value MUST be one of the hash name strings defined in the
+If provided, the "hash_method" value MUST be one of the hash name strings defined in the
 [IANA Named Information Hash Algorithm Registry](https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg).
-
 If the "hash_method" value is not present in the client instance's
 request, the algorithm defaults to "sha-256".
 
-The "sha-256" hash method consists of hashing the input string
+For example, the "sha-256" hash method consists of hashing the input string
```

Comment:
I don't understand why we say it's an example. Here we handle sha-256 and just after, we handle sha3 -> suggestion to remove "for example"

Reply:
Both are actually examples - the definition of the hash function comes from the referenced registry. I'll make them consistent.